models/gmx/arbitrum/glp/gmx_arbitrum_glp_fees.sql

```diff
@@ -21,53 +21,58 @@ GLP fees accrued to the Fee GLP contract and can be claimed by invoking function
 The Fee GLP contract can be found here: https://arbiscan.io/address/0x4e971a87900b931ff39d1aad67697f49835400b6
 */
 
-fglp_balances AS -- This CTE returns the accuals of WETH tokens in the Fee GLP contract and their respective balances in WETH and USD in a designated minute
-    (
-    SELECT  -- This query calculatees the balance in USD of accrued fees ini the Fee GLP contract
-        c.minute,
-        c.balance AS balance_weth
+fglp_balances AS -- This CTE returns the accuals of WETH tokens in the Fee GLP contract in a designated minute
+    (    
+    SELECT -- This subquery aggregates the cumulative balance of WETH tokens in the Fee GLP contract over the minute series
+        b.minute,
+        b.weth_transfer_value,
+        SUM(b.weth_transfer_value) OVER (ORDER BY b.minute ASC) AS weth_cum_balance
     FROM
         (
-        SELECT -- This subquery aggregates the cumulative balance of WETH tokens in the Fee GLP contract over the minute series
-            b.minute,
-            SUM(b.transfer_value) OVER (ORDER BY b.minute ASC) AS balance
+        SELECT  -- This subquery aggregates all the inbound tranfers of WETH tokens to the Fee GLP contract in a designated minute
+            a.minute,
+            SUM(a.transfer_value) AS weth_transfer_value
         FROM
             (
-            SELECT  -- This subquery aggregates all the inbound tranfers of WETH tokens to the Fee GLP contract in a designated minute
-                a.minute,
-                SUM(a.transfer_value) AS transfer_value
-            FROM
-                (
-                SELECT -- This subquery truncates the block time to a minute and selects all the inbound tranfers of WETH tokens to the Fee GLP contract
-                    date_trunc('minute', evt_block_time) AS minute,
-                    ((value) / 1e18) AS transfer_value -- WETH 18dp
-                FROM {{ source('erc20_arbitrum', 'evt_transfer') }}
-                WHERE `to` = '0x4e971a87900b931ff39d1aad67697f49835400b6' -- Fee GLP contract
-                AND `contract_address` = '0x82af49447d8a07e3bd95bd0d56f35241523fbab1' -- WETH Arbitrum Smart Contract
-                ) a
-            GROUP BY a.minute
-            ) b
-        ) c
+            SELECT -- This subquery truncates the block time to a minute and selects all the inbound tranfers of WETH tokens to the Fee GLP contract
+                date_trunc('minute', evt_block_time) AS minute,
+                ((value) / 1e18) AS transfer_value -- WETH 18dp
+            FROM {{ source('erc20_arbitrum', 'evt_transfer') }}
+            WHERE `to` = '0x4e971a87900b931ff39d1aad67697f49835400b6' -- Fee GLP contract
```

Comment:
these could be set as variables and you could share them across all the glp spells https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables